### PR TITLE
licensee 9.14.1 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

`resource` blocks generated using [`generate-rubygems-resources.rb `](https://gist.github.com/SeekingMeaning/4f33f9b233b53818cd88cb1f7f66c2c5)

Previous attempt: https://github.com/Homebrew/homebrew-core/pull/61095

### CODE REVIEW

**Comment by SeekingMeaning** [comment]
https://github.com/Homebrew/homebrew-core/actions/runs/439903395

11.0:

```
libgit2 version is not compatible, expected ~> 1.1.0
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.
```

10.15:

```
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.
```

10.14:

```
mkmf.rb can't find header files for ruby at /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/include/ruby.h
```

File: Formula/licensee.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
+class Licensee < Formula
+  desc "Tool to detect under what license a project is distributed"
+  homepage "https://licensee.github.io/licensee/"
+  url "https://github.com/licensee/licensee/archive/v9.14.1.tar.gz"
+  sha256 "9f90024b4b8de833349c1365ada86657bc6dc25c4cce15c6e5230883a6d39294"
+  license "MIT"
+
+  depends_on "cmake" => :build
```
Comment (dawidd6) [inline Formula/licensee.rb:8]: Where is cmake used?

File: Formula/licensee.rb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  depends_on "libgit2"
+  depends_on "ruby"
```
Comment (dawidd6) [inline Formula/licensee.rb:10]: Did you try uses_from_macos instead?

File: Formula/licensee.rb (hunk 1 above)
Comment (SeekingMeaning) [reply Formula/licensee.rb:8]: I think I added it because a dependency needed it to build a native extension

File: Formula/licensee.rb (hunk 2 above)
Comment (SeekingMeaning) [reply Formula/licensee.rb:10]: Yes, the initial commit had uses_from_macos

**Comment by SeekingMeaning** [comment]
`resource` blocks re-generated using [`generate-rubygems-resources.rb `](https://gist.github.com/SeekingMeaning/4f33f9b233b53818cd88cb1f7f66c2c5)

**Pull request closed without merging**